src/ControllerRunner.ts

```diff
@@ -12,7 +12,7 @@ import {ResponseInterceptorInterface} from "./interceptor/ResponseInterceptorInt
 import {ResponseInterceptorMetadata} from "./metadata/ResponseInterceptorMetadata";
 import {ResultHandleOptions} from "./ResultHandleOptions";
 
-export type Container = { get(someClass: any): any };
+export interface Container { get(someClass: any): any };
```

Comment:
I changed this from a type alias so that custom containers (see [InversifyContainer](https://github.com/codyjs/controllers.ts/blob/inversify/sample/sample7-container/InversifyContainer.ts)) could implement this as an interface.